specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBManagedCassandraDataCenterCreate.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "cassandra-prod-rg",
+    "api-version": "2020-06-01-preview",
+    "clusterName": "cassandra-prod",
+    "dataCenterName": "dc1",
+    "body": {
+      "properties": {
```

Comment:
We need to pass the name here. In the code we are throwing an exception, if the datacenter name in body does not match the URI

Reply:
I think Quetzal was changing this?